test/integration/runonce_duration_test.go

```diff
@@ -0,0 +1,101 @@
+// +build integration
+
+package integration
```

Comment:
With integration tests I like to avoid test tables because then I can't be sure that tests will run without side effects from other tests.